docparse/chunking_strategies.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: 'Chunking Strategies'
+description: 'The chunking strategies supported by Aryn DocParse'
+icon: 'hive'
+---
+
+
+
+When calling DocParse, you can specify a chunking strategy to the `partition_file` call. You can enable the default chunking options by specifying an empty `dict`:
+
+```python
+from aryn_sdk.partition import partition_file
+with open("mydocument.pdf", "rb") as f:
+   data = partition_file(f, chunking_options={})
+```
+
+Here is an example specifying a particular chunking options:
+
+```python
+from aryn_sdk.partition import partition_file
+with open("mydocument.pdf", "rb") as f:
+   data = partition_file(f, 
+      chunking_options={
+         "strategy": "context_rich",
+         "tokenizer": "openai_tokenizer",
+         "tokenizer_options": {
+            "model_name": "text-embedding-3-small"
+         },
+         "merge_across_pages": True,
+         "max_tokens": 512,
+      }
+   )
+```
+
+## Options 
+The options that you can specify in the `dict` include the following: 
+
+ - `strategy`: A string specifying the strategy to use to for chunking. Valid values are `context_rich`, `mixed_multi_column`, and `maximize_within_limit`. The default and recommended chunker is `context_rich` as `{'strategy': 'context_rich'}`.
+        - Behavior of `context_rich` chunker: The goal of this strategy is to add context to evenly-sized chunks. Creates chunks by combining adjacent elements until the chunk reaches the length of the max-token limit specified. Each chunk will contain the copy of the most recently seen section header or title. Section headers or titles that are back to back will be grouped together as one large section header during chunking. 
```

Comment:
Can remove 'evenly-sized', the context_rich chunked doesn't always make sure that the chunks are evenly sized, rather that a chunk is representing a particular section.